src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -177,6 +185,14 @@ function IOUPreview(props) {
         return message;
     };
 
+    const getDisplayAmountText = () => {
```

Comment:
Hi, we didn't handle offline behavior for this function. It defaulted to showing amount as $0.

https://github.com/Expensify/App/issues/29048#issuecomment-1751681314